app/models/record_edit.rb

```diff
@@ -0,0 +1,27 @@
+class RecordEdit < ActiveRecord::Base
+
+  belongs_to :record, polymorphic: true
+  belongs_to :author, class_name: 'User'
+  belongs_to :user, foreign_key: 'author_id' #duplicate author relationship for eager loading via discussion.items
+  has_one    :event, as: :eventable
+
+  validates_presence_of :author, :record
+
+  def self.capture(record, author)
```

Comment:
I dunno, I'm skeptical of putting this into the DB twice. I'd much rather have a VersionSerializer and pull out paper trail with our own implementation when it comes to that, rather than duplicate our edits and have a whole nother table to deal with; we already have all this info saved.